### DISCUSSION TITLE
Valid domain is not accepted in Vercel Project settings

### DISCUSSION THREAD

**Discussion by quifago** [vercel/community#4618] [Help]
### Summary

I own the domain `www.viajandoconniños.com` but when trying to link my Vercel project to my domain, I'm getting the following error:

```
The specified value "www.viajandoconniños.com" is not a fully qualified domain name.
```
<img width="941" alt="image" src="https://github.com/vercel/community/assets/2972550/23249ff8-388f-4a2e-ac2f-43e700133a6c">

I would appreciate some help here... I bought it for 3 years 😭

### Steps to Reproduce

Go to "Project Settings", then "Domains" and try to add `www.viajandoconniños.com` domain on the project

**Comment by amyegan** [comment]
Hey @quifago! It looks like that domain includes special characters. [Punycode](https://www.punycoder.com/) can be used to represent Unicode characters with the limited ASCII character subset that is supported for domains. Please go to this guide for instructions on how to do this: https://vercel.com/guides/how-can-i-use-special-characters-in-my-custom-domain
